src/goats_cli/cli.py

```diff
@@ -33,7 +34,7 @@ def _check_version() -> None:
     utils.display_message("Checking for updates...\n")
     try:
         checker = VersionChecker()
-        if checker.check_if_outdated():
+        if checker.is_outdated:
```

Comment:
You might want to check the output of the errors to see if there were any errors parsing the versions and issue a warning that version checking is being skipped. That can be another ticket in the future.